collab-database/tests/user_test/async_test/script.rs

```diff
@@ -156,21 +146,62 @@ pub async fn run_script(
     },
     DatabaseScript::AssertDatabaseInDisk {
       database_id,
-      expected,
+      expected_fields,
+      expected_rows,
+      expected_view,
     } => {
       let w_database =
         workspace_database_with_db(1, Arc::downgrade(&db), Some(config.clone())).await;
       let database = w_database.get_database(&database_id).await.unwrap();
-      let actual = database.lock().to_json_value();
-      assert_json_diff::assert_json_include!(actual: actual, expected: expected);
+      let database_data = database.lock().get_all_database_data();
+      let view = database.lock().get_view("v1").unwrap();
+
+      assert_eq!(
```

Comment:
using assert_eq! will make the tests hard to maintain. user `assert_json_include` is better